puma/metadata.csv

```diff
@@ -1,6 +1,7 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
 puma.backlog,gauge,,unit,,Pending request backlog,0,puma,request backlog,
 puma.booted_workers,gauge,,unit,,Number of booted puma workers,0,puma,booted workers,
+puma.busy_threads,gauge,,unit,,Number of threads that are running - requests waiting for work + requests waiting on a thread,0,puma,booted workers,
```

Comment:
```suggestion
puma.busy_threads,gauge,,unit,,Number of threads that are running that includes requests waiting for work and requests waiting on a thread.,0,puma,booted workers,
```

Reply:
Looks great, thanks for the suggestion!

Reply:
Actually I had to change this a bit to make sure we're properly reflecting the math. Happy to hear more suggestions on how I could make this more clear.